tests/vm_test.go

```diff
@@ -724,9 +724,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 				Eventually(ThisVMIWith(vm.Namespace, vm.Name), 120).Should(BeRunning())
 
 				By("Ensuring a second invocation should fail")
```

Comment:
what is the motivation for this change? having shorter code? would you state that in the commit message?

Reply:
The motivation is to use a more appropriate matcher for the error. Instead of using ` Expect(err).To(HaveOccurred())` and `Expect(err.Error()).To(Equal())` you can combine it into `Expect(err).To(MatchError()`. But it shortens the code too.